BlueprintUI/Sources/Extensions/BidirectionalCollection+Blueprint.swift

```diff
@@ -0,0 +1,21 @@
+import UIKit
+
+extension BidirectionalCollection where Element: Equatable, Element: NSObjectProtocol {
+    private func itemSearch(_ predicate: UIAccessibilityCustomRotorSearchPredicate) -> UIAccessibilityCustomRotorItemResult? {
+        guard let first else { return nil }
+        guard let currentItem = predicate.currentItem.targetElement as? Element,
```

Comment:
Thinking outloud, im surprised to see both the generic constraint and this check; are both required?

Reply:
The generic constraint is on the array, but the dynamic check is being run on the item that we've been passed in the predicate, which is only guaranteed to be an NSObjectProtocol, not necessarily Equatable. 

We could get away with just checking for Equatable, but I thought it would be nice to ensure that all the types match by using Element instead.